config-scalecube-configuration-service/src/main/java/io/scalecube/config/service/ScalecubeConfigurationServiceConfigSource.java

```diff
@@ -0,0 +1,124 @@
+package io.scalecube.config.service;
+
+import static io.scalecube.services.gateway.clientsdk.Client.http;
+import static io.scalecube.services.gateway.clientsdk.ClientSettings.builder;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.util.MinimalPrettyPrinter;
+import com.fasterxml.jackson.databind.ObjectWriter;
+import io.scalecube.config.ConfigProperty;
+import io.scalecube.config.ConfigSourceNotAvailableException;
+import io.scalecube.config.source.ConfigSource;
+import io.scalecube.config.source.LoadedConfigProperty;
+import io.scalecube.configuration.api.ConfigurationService;
+import io.scalecube.configuration.api.EntriesRequest;
+import io.scalecube.configuration.api.FetchResponse;
+import io.scalecube.services.gateway.clientsdk.ClientSettings.Builder;
+import io.scalecube.services.transport.jackson.JacksonCodec;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.Map;
+import java.util.function.Function;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import reactor.netty.http.HttpResources;
+
+public class ScalecubeConfigurationServiceConfigSource implements ConfigSource {
+
+  private static final Logger LOGGER =
+      LoggerFactory.getLogger(ScalecubeConfigurationServiceConfigSource.class);
+
+  ConfigurationService service;
+
+  private EntriesRequest requestEntries;
+
+  private final Parsing parsing = new Parsing();
+
+  private static URL defaultUrl() {
+    try {
+      return new URL("https", "configuration-service-http.genesis.om2.com", 443, "/");
```

Comment:
Why it's hardcoded? Let's some component (runner) living at the edge  will decide what's _default_  for him.